pkg/kcp/provider/gcp/redisinstance/removeReadyCondition.go

```diff
@@ -22,7 +23,16 @@ func removeReadyCondition(ctx context.Context, st composed.State) (error, contex
 	logger.Info("Removing Ready condition")
 
 	meta.RemoveStatusCondition(redisInstance.Conditions(), cloudcontrolv1beta1.ConditionTypeReady)
-	err := state.UpdateObjStatus(ctx)
+
+	serializedJson, err := json.Marshal(redisInstance.Status)
```

Comment:
Let's see what exactly will get sent
```suggestion
	serializedJson, err := json.Marshal(redisInstance.CloneForPatchStatus().Status)
```